projects/packages/videopress/src/client/block-editor/hooks/use-auto-generated-chpater.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import debugFactory from 'debug';
+/**
+ * WordPress dependencies
+ */
+import getChapter from '../../lib/get-chapter';
+import { VideoChapter } from '../../lib/get-chapter/types';
+import { VideoGUID } from '../blocks/video/types';
+/**
+ * Internal dependencies
+ */
```

Comment:
Seems we don't have internal dependencies imported in this file so we could remove this comment block.